android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java

```diff
@@ -179,6 +181,19 @@ private void displaySnackbar(View view, ReadableMap options, final Callback call
         TextView snackbarText = snackbarView.findViewById(com.google.android.material.R.id.snackbar_text);
         snackbarText.setMaxLines(numberOfLines);
         snackbarText.setTextColor(textColor);
+        if (textAlignCenter) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M){
+                snackbarText.setTextAlignment(View.TEXT_ALIGNMENT_CENTER);
+            } else {
+                snackbarText.setGravity(Gravity.CENTER_HORIZONTAL);
+            }
+        } else {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M){
+                snackbarText.setTextAlignment(View.TEXT_ALIGNMENT_TEXT_START);
+            } else {
+                snackbarText.setGravity(Gravity.START);
```

Comment:
My only concern is that this runs new code by default where it did nothing before, which could subtly alter how it looks on certain devices. I assume it won't change anything.